pkg/shell/shell.go

```diff
@@ -134,7 +134,13 @@ func GetShell(userName string) ([]string, error) {
 			return []string{shell}, nil
 		}
 
-		// fallback to path discovery if unsuccessful
+		// fallback to $SHELL env var
+		shell, ok := os.LookupEnv("SHELL")
```

Comment:
could move to a one liner

Reply:
will fix the next time we touch this code if you don't mind